taqueria-analytics/index.ts

```diff
@@ -30,20 +33,25 @@ export const inject = ({ taqVersion, taqBuild, fields, getMachineId, operatingSy
 		events.push({
 			name,
 			params: {
-				...params,
-				...fields,
+				taq_error: false,
```

Comment:
What if there is an error? Is the error case handled elsewhere?

Reply:
Yes, this is setting taq_error to false a default value. It will get overridden in params when a consumer of this package calls `addEvent`.